### ISSUE TITLE
Eslint@5.9.0 complains about specific plugins I don't know about

### ISSUE THREAD

**Issue by brunoscopelliti** [eslint/eslint#11126]
**Tell us about your environment**

* **ESLint Version:** 5.9.0
* **Node Version:** 10.2.1
* **npm Version:** 5.6.0

**What parser (default, Babel-ESLint, etc.) are you using?**
Default.

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
  "extends": "standard",
  "rules": {
    "comma-dangle": ["error", {
      "arrays": "always-multiline",
      "objects": "always-multiline",
      "imports": "always-multiline",
      "exports": "always-multiline",
      "functions": "ignore"
    }],
    "curly": ["error", "all"],
    "eqeqeq": ["error", "smart"],
    "no-extra-semi": "error",
    "quotes": ["error", "double"],
    "semi": ["error", "always"]
  },
  "globals": {
    "DocumentFragment": true,
    "fetch": true,
    "HTMLElement": true,
    "location": true,
    "Node": true
  }
}
```
</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```bash
npx eslint src/**
```

**What did you expect to happen?** I expected eslint would work as usual.

**What actually happened? Please include the actual, raw output from ESLint.**

I got this error.

```bash
Failed to load plugin ie11: Cannot find module 'eslint-plugin-ie11'
```

I tried to install eslint-plugin-ie11, then it complained about eslint-plugin-local-rules. When I installed it, it broke with an exception about a file in eslint-plugin-local-rules, that cannot be found.

Please note that my configuration extend `standard`. I never heard about those plugins. Any help would be appreciated.

**Comment by nzakas** [comment]
Did you setup your configuration file using `--init` or some other way?

If you use `--init`, it should install all the dependencies automatically. If you installed it manually, then you need to follow these instructions:
https://github.com/standard/eslint-config-standard#usage

**Comment by brunoscopelliti** [comment]
Hi @nzakas .
No I didn't use `--init`.
I'll read the doc in the coming days, and come back with a feedback.
Anyway I was surprised by having this issue, cause eslint worked just fine before I updated to latest version (previous version was 5.8.0).
In the meanwhile, thank you for your help.

**Comment by brunoscopelliti** [comment]
Hi @nzakas 

I've repeated the configuration, following the instructions at the link you've posted, but the results don't change. I still get the same error about the eslint ie11 plugin.

**Comment by nzakas** [comment]
@brunoscopelliti at this point, it's probably best for you to file an issue with [`eslint-config-standard`](https://github.com/standard/eslint-config-standard) as there must be some deep dependency not being met in that package.

**Comment by brunoscopelliti** [comment]
Thanks @nzakas.
I'll try to setup a simple repro, and open an issue on eslint-config-standard in the coming days.
Thank you for your feedback.

**Comment by brunoscopelliti** [comment]
I've some updates.

I've tried to create a smaller _repro_ of the issue, but I wasn't able to reproduce it in another new project.
But I've still the original issue.

What I've tried:
* Delete, and reinstall dependencies.
* Delete global eslint installation.
* Upgrade node.js (current version 11.3.0).

Nothing worked so far.

A colleague of mine tried to on his computer (node v6.*) and the command executed successfully.

@nzakas Do you know if there's some kind of cache eslint checks I should be aware of?

**Comment by brunoscopelliti** [comment]
I've finally understood the problem.
This is the project folder structure

```
<root>
    - index.js
    - project-a
       - package.json
       - node_modules <--- caused the problem 
```

To solve the problem I added _node_modules_ in eslintignore file.
